Jet_fitting_3/doc/Jet_fitting_3/CGAL/Monge_via_jet_fitting.h

```diff
@@ -199,7 +199,7 @@ Monge_via_jet_fitting();
 This operator performs all the computations. The \f$ N\f$ input points are 
 given by the `InputIterator` parameters which value-type are 
 `Data_kernel::Point_3`, `d` is the degree of the fitted 
-polynomial, `d'` is the degree of the expected Monge 
+polynomial, \c d' is the degree of the expected Monge
```

Comment:
I guess it should be `\f$ d' \f$` here.

Reply:
Is also a possibility, but I changed in a way that is compatible with the old situation.